dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -465,6 +465,46 @@ public void TipsWhichDoNotFitDoNotOpen()
             }
         }
 
+        [TestMethod]
+        public void VerifyTheming()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
+                var actionButtonComboBox = elements.GetActionButtonContentComboBox();
+                actionButtonComboBox.SelectItemByName("Small text");
+                elements.GetSetActionButtonContentButton().Invoke();
```

Comment:
I've put actions like this into helper functions, I think it makes the test easier to read at a glance.